includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "http://fandom.wikia.com",
+				"image" => "wds/full_fandom_logo",
+				"title" => [
+					"type" => "text",
+					"value" => "Fandom",
+				]
+			]
+		],
+		"verticals" => [
+			"links" => [
+				[
+					"type" => "link-branded",
+					"brand" => "tv",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-tv",
+					],
+					"href" => "http://tv.wikia.com"
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "games",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-games",
+					],
+					"href" => "http://games.wikia.com",
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "movies",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-movies",
+					],
+					"href" => "http://movies.wikia.com",
+				],
+			],
+		],
+		"wikis" => [
+			"header" => [
+				"type" => "link-branded",
+				"brand" => "wikis",
+				"href" => "#",
+				"title" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-brandlink-vertical-wikis",
+				],
+			],
+			"links" => [
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-explorewikis",
+					],
+					"href" => "#",
+				],
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-wikis-communitycentral",
+					],
+					"href" => "#",
+				],
+			],
+		],
+		"search" => [
+			"module" => [
+				"type" => "search",
+				"results" => [
+					"url" => "http://wikia.com/search",
+					"param" => "query",
+				],
+				"suggestions" => [
+					"url" => "http://wikia.com/search/suggestions",
+					"param" => "query",
+				],
+				"placeholder-inactive" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-search-placeholder-inactive",
+				],
+				"placeholder-active" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-search-placeholder-active",
+				],
+			],
+		],
+		"anon" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "#",
+				"image" => "anon-avatar",
```

Comment:
For logo we used `wds` prefix. Can we change here or there to be more consistent?